tests/data/test_dataloader.py

```diff
@@ -428,27 +428,102 @@ def test_tokenize_example_malformed():
         'response': 'response',
         'completion': 'completion'
     }
+    no_content = {'messages': [{'role': 'user'}]}
+    ends_with_user_role: ChatFormattedDict = {
```

Comment:
does pyright complain if you don't have this type hint `ChatFormattedDict` here?

Reply:
yes I think so

Reply:
can you double check? pyright seems ok on my end if i remove them. I just want to make sure we're not introducing any limitations with type aliases. (Otherwise, type aliases are awesome!)

Reply:
ah good catch, I removed and pyright did not complain